demo/src/App.tsx

```diff
@@ -145,13 +148,14 @@ const App = () => {
         const startTime = performance.now();
         const results = await model.run(feeds);
         const endTime = performance.now();
-        const executionTime = endTime - startTime;
-        setExecutionTime(executionTime);
-        console.log(`model.run() took ${executionTime} ms`);
+        const eT = endTime - startTime;
```

Comment:
Odd change. `executionTime` is a much clearer variable name than `eT`? (I'm still merging with it, but may change it back afterward)